review/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class Review(models.Model):
+    review = models.TextField()
+    dogowner_id = models.ForeignKey(DogOwner, on_delete=models.DO_NOTHING)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    rating = models.IntegerField(validators=[MinValueValidator(0), MaxValueValidator(5)])
+    creation_date = models.DateTimeField(auto_now_add=True)
+
+
+@receiver(post_save, sender=Review)
+def save_user_profile(sender, instance, **kwargs):
+    if instance.full_clean():
+        instance.save()
```

Comment:
I think signals is more clean solution here, when you use the model it will be like nothing was changed

Reply:
A more simple solution would be to define static create method that also call save(),
it will also simplify the tests, and consistent with other models.

In general I find using signals just for persisting an object misuse.
Moreover I saw that save() is called on tests, if it doesn't work w/o it 
I am not sure  what is the value of having `save_user_profile`.

Reply:
The purpose of this function is to run the full clean (e.g. validators that were defined) and save the object it it passed.
In this case we just run the regular save() and the signals calling this function and running the validation.
We can also create static function that we will the same but then we need to make sure to use it instead of the save directly and in case someone will call save directly the object will be created without the validations

Reply:
Validations should be part of the object creation flow, if the arguments are not valid the object should not be created.
In this case you can define static create method / constructor with the validation and persistence logic.
This way it is guaranteed that a created object is valid and persisted, w/o using signal magics.